osmaxx-py/excerptconverter/tests/test_gis_excerpt_converter.py

```diff
@@ -98,7 +98,8 @@ def test_create_output_file(self):
         creation_status = gis_excerpt_converter.create_output_file(
             self.extraction_order,
             self.result_file.name,
-            'gpkg'
+            'gpkg',
+            ''
```

Comment:
This is the name of the directory inside the result media storage. For testing it doesn't care where the files will be created. It will just inflate the test to use a subdirectory here.